src/myframe.h

```diff
@@ -278,6 +278,7 @@ struct MyFrame : wxFrame {
             selmenu->AppendSeparator();
             MyAppend(selmenu, A_LINK, _(L"Go To &Matching Cell\tF6"));
             MyAppend(selmenu, A_LINKREV, _(L"Go To Matching Cell (Reverse)\tSHIFT+F6"));
+            MyAppend(selmenu, A_LINKIMG, _(L"Go To Cell with Matching Image\tSHIFT+F7"));

```

Comment:
that's a bit of a weird keyboard assignment. Why don't you make `L"Toggle Vertical Layout\tF7"` into `ALT+0`, then you can make the image search into `F7` (and have room to add a reverse in the future if anyone needs it)

Reply:
Thanks for the review and the helpful input! :+1: 
I revised the PR accordingly. I hope this fits better now.